code-samples/creating-your-bot/event-handling/index.js

```diff
@@ -16,11 +24,11 @@ for (const file of eventFiles) {
 	if (event.once) {
 		client.once(event.name, (...args) => event.execute(...args));
 	} else {
-		// client.on(event.name, (...args) => event.execute(...args));
-		client.on(event.name, (oldMember, newMember) =>
-			event.execute(oldMember, newMember)
-		);
+		client.on(event.name, (...args) => event.execute(...args));
+		//client.on(event.name, (oldMember, newMember) =>
```

Comment:
why commented out?

Reply:
It's redundant, does the same as line 27
`client.on(event.name, (...args) => events.execute(...args));`

Will take it out in final release